test_caseinsensitivedict.py

```diff
@@ -67,6 +67,16 @@ def test_copy(self):
         #     Return the value for key if key is in the dictionary, else
         #     default.  If default is not given, it defaults to None, so that
         #     this method never raises a KeyError.
+    def test_get(self):
+        myinstance = CaseInsensitiveDict()
+        myinstance['Kaisa'] = 5
+        self.assertIn('Kaisa', myinstance)
+        self.assertNotIn('Jhin', myinstance)
+        value = myinstance.get('Kaisa')
```

Comment:
add a case where we `.get()` the string 'kaisa' (all lower case). it should work for a case-insensitive dictionary and in this case it should return 5.